src/Symfony/Component/Cache/Adapter/TraceableAdapter.php

```diff
@@ -22,7 +22,8 @@
  */
 class TraceableAdapter implements AdapterInterface
 {
-    private $pool;
+    /** @internal */
```

Comment:
let's remove that (and the one on the start method)
they are part of the internal api/respo of the class